src/js/components/Select/Select.js

```diff
@@ -172,7 +172,8 @@ const Select = forwardRef(
     const onRequestClose = useCallback(() => {
       setOpen(false);
       if (onClose) onClose();
-    }, [onClose]);
+      setSearch();
+    }, [onClose, setSearch]);
```

Comment:
```suggestion
    }, [onClose]);
```
I don't think we need to include setSearch in the dependencies array since the setSearch itself isn't changing